integration-cli/docker_cli_rmi_test.go

```diff
@@ -170,17 +137,16 @@ func (s *DockerSuite) TestRmiImageIDForceWithRunningContainersAndMultipleTags(c
 	testRequires(c, DaemonIsLinux)
 	dockerfile := "FROM busybox\nRUN echo test 14116\n"
 	imgID, err := buildImage("test-14116", dockerfile, false)
-	c.Assert(err, check.IsNil)
+	c.Assert(err, checker.IsNil)
 
 	newTag := "newtag"
 	dockerCmd(c, "tag", imgID, newTag)
 	dockerCmd(c, "run", "-d", imgID, "top")
 
 	out, _, err := dockerCmdWithError("rmi", "-f", imgID)
-	if err == nil || !strings.Contains(out, "(cannot be forced) - image is being used by running container") {
-		c.Log(out)
-		c.Fatalf("rmi -f should not delete image with running containers")
-	}
+	// rmi -f should not delete image with running containers
+	c.Assert(err, checker.NotNil)
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?